vllm/entrypoints/context.py

```diff
@@ -161,6 +172,16 @@ async def call_python_tool(self, tool_session: Union["ClientSession",
                     recipient=Role.ASSISTANT)
         ]
 
+    async def init_tool_sessions(self, tool_server: Optional[ToolServer],
+                                 exit_stack: AsyncExitStack) -> None:
+        if tool_server:
+            for tool_name, session in self._tool_sessions.items():
+                if isinstance(session, str):
```

Comment:
Would it be cleaner if you also initialize tool_sessions for `DemoToolServer` here?